mne/channels/montage.py

```diff
@@ -312,7 +312,8 @@ class DigMontage(object):
     Parameters
     ----------
     hsp : array, shape (n_points, 3)
-        The positions of the channels in 3d.
+        The positions of the headshape points in 3d.
+        These points are in the acquisition digitizer space.
```

Comment:
...and there is code that uses `head_dev_t` to transform these points to `FIFFV_COORD_DEVICE`...